workflows/nimble_sdk_helper.py

```diff
@@ -1,13 +1,18 @@
 import sys
```

Comment:
Please add copyright header to source files

Reply:
Done!